day-1/examples/Files.ipynb

```diff
@@ -49,24 +49,24 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 3,
+   "execution_count": 7,
    "metadata": {},
    "outputs": [
     {
      "data": {
       "text/plain": [
-       "<matplotlib.axes._subplots.AxesSubplot at 0x7fa5a1918c50>"
```

Comment:
I do nit thin, we have to change this line. Because it makes it hard for other people to merge teir chenges.

Reply:
I agree. I will create a development branch, and probably make the changes there. Is that recommended?